### DISCUSSION TITLE
posts/the-defect-of-node/

### DISCUSSION THREAD

**Discussion by giscus** [yunnysunny/yunnysunny.github.io#14] [Q&A]
# posts/the-defect-of-node/

提起 Node.js ，大家可能认为对其的优劣已经分辨的很清楚：由于其默认单线程，所以不适合做计算密集型操作；由于其异步 IO 特性，所以对于 IO 处理比较友好。但这其实是一种误解，当 IO 并发到一定的量级，一样会损耗 CPU 资源。在 Node.js 引以为豪的 IO 处理方面，依然有其做的不完善的地方，本篇文章就给大家一一解密。

https://blog.whyun.com/posts/the-defect-of-node/

**Comment by sinianzhiren** [comment]
大佬，图 1.3.2 这是什么监测工具，开源的？有地址吗

**Reply by yunnysunny** [reply]
alinode平台提供的，具体参见教程 https://blog.whyun.com/nodebook#/11_node_optimization?id=_114-alinode
